### ISSUE TITLE
Failed to register native method 'freerdp_send_unicodekey_event'

### ISSUE THREAD

**Issue by amratsingh** [FreeRDP/FreeRDP#5368]
After setting up FreeRDP and executing shell command I was able to generate jni files. Now I trying to run android project (Studio) but app crashes every time when I open it. I think a method named 'freerdp_send_unicodekey_event' creating problem. Logcat have long list of lines but I am posting relevant main error lines here:

```
com.freerdp.afreerdp E/zygote64: Failed to register native method com.freerdp.freerdpcore.services.LibFreeRDP.freerdp_send_unicodekey_event(JI)Z in /data/app/com.freerdp.afreerdp-MdA66nM6dFZKtRxswGz-3g==/base.apk!classes5.dex
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534] JNI DETECTED ERROR IN APPLICATION: JNI NewGlobalRef called with pending exception java.lang.NoSuchMethodError: no static or non-static method "Lcom/freerdp/freerdpcore/services/LibFreeRDP;.freerdp_send_unicodekey_event(JI)Z"
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader, java.lang.String) (Runtime.java:-2)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at java.lang.String java.lang.Runtime.doLoad(java.lang.String, java.lang.ClassLoader) (Runtime.java:1099)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at void java.lang.Runtime.loadLibrary0(java.lang.ClassLoader, java.lang.String) (Runtime.java:1014)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at void java.lang.System.loadLibrary(java.lang.String) (System.java:1657)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at void com.freerdp.freerdpcore.services.LibFreeRDP.<clinit>() (LibFreeRDP.java:44)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at void com.freerdp.freerdpcore.services.LibFreeRDP.setEventListener(com.freerdp.freerdpcore.services.LibFreeRDP$EventListener) (LibFreeRDP.java:97)
com.freerdp.afreerdp A/zygote64: java_vm_ext.cc:534]   at void com.freerdp.freerdpcore.application.GlobalApp.onCreate() (GlobalApp.java:123)
```

**Comment by akallabeth** [comment]
Please do not open issues for support questions. Use the mailing list or irc for such.

(Just tried the current vanilla android client, that one works)

**Comment by amratsingh** [comment]
Would have been good if you had answered my question also, as you answerd [here](https://github.com/FreeRDP/FreeRDP/issues/5369#issuecomment-488554985). I had a same problem. It is solved with [#5371](https://github.com/FreeRDP/FreeRDP/pull/5371) merge.

**Comment by akallabeth** [comment]
@amratsingh sorry, looks like that was my fault, tested with a new build but it looks like I was using the wrong branch :/
